src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {

```

Comment:
It's better to create Interface 'Drawable' and implement method 'draw()' in Circle and other figures. Don't implement 'draw()' in this abstract class
![image](https://user-images.githubusercontent.com/117547096/218504878-130447d2-8237-4a11-bd5b-968bd4fcf7e5.png)
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```